definitions-terms.md

```diff
@@ -0,0 +1,108 @@
+# Definitions and Terms
+
+A collection of definitions and terms used within this repository.
+
+## TOC
+
+* [Fingerprint](#fingerprint)
+* [Fully Qualified Artifact Reference](#registry-fully-qualified-reference)
+* [Multi-Tenant Registry](#multi-tenant-registry)
+* [OCI Annotations](#oci-annotations)
+* [OCI Descriptor](#oci-descriptor)
+* [OCI Index](#oci-index)
+* [OCI Manifest](#oci-manifest)
+* [Registry](#registry)
+* [SBoM](#sbom)
+* [Subject](#subject)
+* [Tag](#tag)
+
+## Artifact / Image
+
+Artifacts are a generalization of how images are stored in an instance of the [OCI Distribution Spec][oci-distribution-spec]. An [OCI Image][oci-image] is a type of [OCI Artifact][oci-artifact]. When referencing artifacts, within a [fully qualified reference](#registry-fully-qualified-reference), the artifact is right most element of the namespace, combined with the `:tag`.  
+Reference: [OCI Artifacts][oci-artifact]  
+![](./media/artifact-ref.png)  
+
+
+## Fingerprint
+
+A short identifier of a given public key.  
+Reference: [Fingerprint][fingerprint]
+
+## Multi-Tenant Registry
```

Comment:
I'd vote for excluding this definition for now.  This is getting a little slippery with codifying the practice of registry operators instead of defining a term that we intend to use going forward.

Reply:
How do we articulate the difference between a unique registry, and the registry as a whole? What is "your registry" on docker hub, or within AWS? Just think we need to articulate the difference.

Reply:
I'd vote on holding off on articulating the difference at all until we need it.  I'm not convinced that it's really necessary for the discussion.

Reply:
After reviewing the registry definition, it already covered unique orgs by namespace or domain. This basically covers the thing I was trying to cover for multi-tenant registries.